api/HTMLScriptElement.json

```diff
@@ -169,10 +169,10 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "4"
+              "version_added": "6"
```

Comment:
Can you check where the original "4" came from? Perhaps the content attribute was supported earlier?